doc: consistent use of word child process (#55654)
nodejs/node: doc/api/child_process.md

> If `timeout` is greater than `0`, the parent will send the signal

Reviewer: ```suggestion
If `timeout` is greater than `0`, the parent process will send the signal
```

Missed one.

    AbortSignal.
  * `killSignal` {string|integer} The signal value to be used when the spawned
    process will be killed by timeout or abort signal. **Default:** `'SIGTERM'`.
  * `silent` {boolean} If `true`, stdin, stdout, and stderr of the child

>     process will be piped to the parent, otherwise they will be inherited

Reviewer 2: Shouldnt we Apple the same for parent process?
```suggestion
    process will be piped to the parent process, otherwise they will be inherited
```

>     from the parent, see the `'pipe'` and `'inherit'` options for

Reviewer 2: ```suggestion
    from the parent process, see the `'pipe'` and `'inherit'` options for
```

-->

On Windows, setting `options.detached` to `true` makes it possible for the
child process to continue running after the parent exits. The child process
will have its own console window. Once enabled for a child process,
it cannot be disabled.

On non-Windows platforms, if `options.detached` is set to `true`, the child
process will be made the leader of a new process group and session. Child
processes may continue running after the parent exits regardless of whether
they are detached or not. See setsid(2) for more information.

By default, the parent will wait for the detached child process to exit.

> To prevent the parent from waiting for a given `subprocess` to exit, use

Reviewer 2: ```suggestion
To prevent the parent process from waiting for a given `subprocess` to exit, use
```

> the `subprocess.unref()` method. Doing so will cause the parent's event
> loop to not include the child process in its reference count, allowing the
> parent to exit independently of the child, unless there is an established

Reviewer 2: There are still a few occurences
```suggestion
the `subprocess.unref()` method. Doing so will cause the parent process' event
loop to not include the child process in its reference count, allowing the
parent process to exit independently of the child process, unless there is an established
```

IPC channel between the child and the parent processes.

When using the `detached` option to start a long-running process, the process
will not stay running in the background after the parent exits unless it is
provided with a `stdio` configuration that is not connected to the parent.

> If the parent's `stdio` is inherited, the child process will remain attached

Reviewer 2: ```suggestion
If the parent process' `stdio` is inherited, the child process will remain attached
```

```

Child Node.js processes will have a [`process.send()`][] method of their own

> that allows the child process to send messages back to the parent.

Reviewer 2: ```suggestion
that allows the child process to send messages back to the parent process.
```

added: v0.7.10
-->

> By default, the parent will wait for the detached child process to exit.
> To prevent the parent from waiting for a given `subprocess` to exit, use the

Reviewer 2: ```suggestion
By default, the parent process will wait for the detached child process to exit.
To prevent the parent process from waiting for a given `subprocess` to exit, use the
```
